cluster-scope/overlays/prod/moc/infra/kustomization.yaml

```diff
@@ -24,7 +24,6 @@ resources:
   - ../../../../base/operators.coreos.com/subscriptions/cert-manager
   - ../../../../base/operators.coreos.com/subscriptions/cluster-logging
   - ../../../../base/operators.coreos.com/subscriptions/kubernetes-nmstate-operator
-  - ../../../../base/operators.coreos.com/subscriptions/rhsso-operator
```

Comment:
Please remove `operatorgroup/keycloak` and `namespaces/keycloak` as well. Those can be also removed from `cluster-scope/base`